### PR TITLE
[site] Added pci ssc block on main and security pages

### PR DESCRIPTION
**Pull request by ToyTox**

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

File: docs/site/_data/modal.yml [hunk 1]
```diff
@@ -64,3 +64,28 @@ request-education:
...
       title: "Error"
       text: "Something went wrong..."
+get-a-report:
+  header:
+    ru:
+      title: "Получите отчет о&nbsp;соответствии рекомендациям PCI&nbsp;SSC"
+      text: "Документ ориентирован на&nbsp;компании, которые используют платежные системы на&nbsp;базе контейнеров, и&nbsp;помогает защитить эти системы от&nbsp;актуальных угроз. Мы&nbsp;выяснили, насколько Deckhouse соответствует критериям PCI&nbsp;SSC."
+    en:
+      title: "Get a report"
+      text: "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Libero, magnam!"
```
Comment (z9r5) [inline docs/site/_data/modal.yml:74]: @ToyTox TODO?

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @ToyTox. I've started the workflow for label 'deploy/web/stage'...

**Review by z9r5 — APPROVED** [review]

**Pull request merged**